test/functional/apps/discover/_discover_histogram.js

```diff
@@ -35,6 +36,11 @@ export default function({ getService, getPageObjects }) {
     before(async function() {
       log.debug('load kibana index with default index pattern');
       await PageObjects.common.navigateToApp('home');
+      await security.testUser.setRoles([
```

Comment:
Missing `.restoreDefaults()` call in this suite.

Reply:
added